index.php

```diff
@@ -46,6 +46,36 @@
 define('NO_UPGRADE_CHECK', true);
 define('ABORT_AFTER_CONFIG', true);
 
+/**
+ * Checks if the command line maintenance mode has been enabled. Skip the config bootstrapping.
+ *
+ * @param string $configfile The relative path for config.php
+ * @return bool True if climaintenance.html is found.
+ */
+function checkclimaintenance($configfile) {
+    $content = file_get_contents($configfile);
+    $content = preg_replace("/[^!:]\/\//", "\n//", $content);  // Set comments to be on newlines, replace '//' with '\n//', where // does not start with :
```

Comment:
Using another regex delimiter could make it more readable:
`preg_replace("#[^!:]//#", "\n//", $content);`